packages/client/src/client/Web/sw.js

```diff
@@ -83,3 +84,27 @@ self.addEventListener("message", (event) => {
     self.skipWaiting()
   }
 })
+
+// from https://web.dev/service-worker-lifecycle/#skip-the-waiting-phase
+self.addEventListener("install", (event) => {
+  console.debug("SW: install", event)
+  // self.skipWaiting()
```

Comment:
I suppose the reason we are not calling `skipWaiting` here is that we want to show the reload dialog?